src/alg_utils.jl

```diff
@@ -73,7 +84,9 @@ alg_extrapolates(alg::IRKN3) = true
 alg_extrapolates(alg::ABDF2) = true
 
 alg_order(alg::OrdinaryDiffEqAlgorithm) = error("Order is not defined for this algorithm")
+alg_order(alg::CompositeAlgorithm) = (alg_order.(alg.algs)...)
 alg_adaptive_order(alg::OrdinaryDiffEqAdaptiveAlgorithm) = error("Algorithm is adaptive with no order")
+alg_adaptive_order(alg::CompositeAlgorithm) = alg_adaptive_order(alg.algs[alg.current_alg])
```

Comment:
this'll not be thread-safe. `current_alg` shouldn't be used, in fact it should probably just be removed here. Instead, the internals of the adaptive algorithm should have a `get_current_adaptive_order(alg,cache)` that returns the current order (for most, it's just `alg_adaptive_order`, and `CompositeAlgorithm` gets a dispatch that it uses the current from the cache, not the alg).